storage/dvo_recommendations_storage.go

```diff
@@ -357,6 +357,15 @@ func (storage DVORecommendationsDBStorage) updateReport(
 		}
 
 		values[8] = lastCheckedTime // last_checked_at
+
+		ruleHitsCountsJSON, err := json.Marshal(ruleHitsCounts[namespaceUID])
```

Comment:
please cooperate with @joselsegura to unify the approach of writing/reading the json column

Reply:
I modified writing approach according to @joselsegura PR.